benchmark/kvbench/commands/nixlbench.py

```
@@ -206,6 +218,20 @@ def _configure_posix(self, source: str, destination: str):
         else:
             raise ValueError(f"Invalid source for POSIX/HF3FS: {source}")
 
+    def _configure_spdk(self, source: str, destination: str):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_configure_spdk`

Add return type annotation: `None`

(ANN202)